### PR TITLE
Flip the sample app to be less quiet.

### PR DESCRIPTION
**Pull request by dj2**
The sample app outputting just an error code by default is confusing to
users. The -s flag was originally created to turn on the summary mode.
This CL removes -s from the help (but it still parses) and adds a -q
flag instead. The quiet flag is used by the test runner to suppress the
summary output.

### CODE REVIEW

**Comment by dj2** [comment]
@alan-baker

File: samples/amber.cc [hunk 1]
```diff
@@ -147,8 +148,11 @@ bool ParseArgs(const std::vector<std::string>& args, Options* opts) {
       opts->parse_only = true;
     } else if (arg == "-d") {
       opts->disable_validation_layer = true;
+    // -s is an old option which is still recognized but does nothing.
     } else if (arg == "-s") {
       opts->show_summary = true;
```
Comment (alan-baker) [inline samples/amber.cc]: Maybe remove `show_summary` (since it seems unused) and make this `quiet = false` (or ignore)?

Comment (dj2) [reply samples/amber.cc]: good point, done.

File: samples/amber.cc [hunk 2]
```diff
@@ -151,8 +151,11 @@ bool ParseArgs(const std::vector<std::string>& args, Options* opts) {
       opts->pipeline_create_only = true;
     } else if (arg == "-d") {
       opts->disable_validation_layer = true;
+      // -s is an old option which is still recognized but does nothing.
```
Comment (dneto0) [inline samples/amber.cc]: Well, it does something: It reverses the -q option...

Comment (dj2) [reply samples/amber.cc]: done

**Pull request merged**